rules/manager.go

```diff
@@ -818,17 +824,28 @@ func (m *Manager) AlertingRules() []*AlertingRule {
 
 // Describe implements prometheus.Collector.
 func (m *Manager) Describe(ch chan<- *prometheus.Desc) {
+	ch <- lastEvaluation
 	ch <- lastDuration
 	ch <- groupInterval
 }
 
 // Collect implements prometheus.Collector.
 func (m *Manager) Collect(ch chan<- prometheus.Metric) {
 	for _, g := range m.RuleGroups() {
+		lastEvaluationTime := g.GetEvaluationTimestamp()
+		lastEvaluationTimestamp := 0.0
```

Comment:
I think `NaN` or `-Inf` would make more sense than 0 here. If I think about how I'd use it in an alert (alerting on a group not having been evaluated in a too long time), my expression would be:

    time() - prometheus_ rule_group_last_evaluation_timestamp_seconds > x

...combined with a `for ...` clause. In that scenario, `-Inf` would be most useful, as I wouldn't have to add another check for it staying in `NaN` for too long. @brian-brazil agreed?

Reply:
`-Inf` looks more useful.